### PR TITLE
Created New Stable API Version for python3 Runbook APIs

### PR DESCRIPTION
**Pull request by sushil490023**
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i>

### Changelog
Add a changelog entry for this PR by answering the following questions:
  1. What's the purpose of the update?
      - [ ] new service onboarding
      - [x] new API version
      - [ ] update existing version for new feature
      - [ ] update existing version to fix swagger quality issue in s360
      - [ ] Other, please clarify
  2. When are you targeting to deploy the new service/feature to public regions? Please provide the date or, if the date is not yet available, the month.
  Its available in all public regions
  
  4. When do you expect to publish the swagger? Please provide date or, the the date is not yet available, the month.
This is PR to update swagger.
  6. If updating an existing version, please select the specific language SDKs and CLIs that must be refreshed after the swagger is published.
      - [ ] SDK of .NET (need service team to ensure code readiness)
      - [x] SDK of Python
      - [ ] SDK of Java
      - [ ] SDK of Js
      - [ ] SDK of Go
      - [ ] PowerShell
      - [ ] CLI
      - [ ] Terraform
      - [ ] No refresh required for updates in this PR

### Contribution checklist:
- [x] I commit to follow the [Breaking Change Policy](http://aka.ms/bcforapi) of "no breaking changes"
- [x] I have reviewed the [documentation](https://aka.ms/ameonboard) for the workflow.
- [ ] [Validation tools](https://aka.ms/swaggertools) were run on swagger spec(s) and errors have all been fixed in this PR. [How to fix?](https://aka.ms/ci-fix)

If any further question about AME onboarding or validation tools, please view the [FAQ](https://aka.ms/faqinprreview).

### ARM API Review Checklist 

> **Applicability**: :warning: 
>
> If your changes encompass only the following scenarios, you should SKIP this section, as these scenarios do not require ARM review.
> - Change to data plane APIs 
> - Adding new properties 
> - All removals

Otherwise your PR may be subject to ARM review requirements. Complete the following:
- [x] Check this box if any of the following appy to the PR so that the label "ARMReview" and "WaitForARMFeedback" will be added by bot to kick off ARM API Review.  Missing to check this box in the following scenario may result in delays to the ARM manifest review and deployment.
  - Adding a new service
  - Adding new API(s)
  - Adding a new API version
    -[ ] To review changes efficiently, ensure you are using OpenAPIHub to initialize the PR for adding a new version. More details, refer to the [wiki](https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/208/OpenAPI-Hub-Adding-new-API-version).

- [ ] Ensure you've reviewed following [guidelines](https://aka.ms/rpguidelines) including [ARM resource provider contract](https://github.com/Azure/azure-resource-manager-rpc) and [REST guidelines](https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md). Estimated time (4 hours). This is required before you can request review from ARM API Review board.

- [ ] If you are blocked on ARM review and want to get the PR merged with urgency, please get the ARM oncall for reviews (*RP Manifest Approvers* team under <ins>Azure Resource Manager service</ins>) from IcM and reach out to them. 

### Breaking Change Review Checklist 
If any of the following scenarios apply to the PR, request approval from the Breaking Change Review Board as defined in the [Breaking Change Policy](http://aka.ms/bcforapi). 
- [ ] Removing API(s) in a stable version
- [ ] Removing properties in a stable version
- [ ] Removing API version(s) in a stable version
- [ ] Updating API in a stable or public preview version with Breaking Change Validation errors
- [ ] Updating API(s) in public preview over 1 year (refer to [Retirement of Previews](https://dev.azure.com/msazure/AzureWiki/_wiki/wikis/AzureWiki.wiki/37683/Retirement-of-Previews))

**Action**: to initiate an evaluation of the breaking change, create a new intake using the [template for breaking changes](https://aka.ms/Breakingchangetemplate). Addition details on the process and office hours are on the [Breaking change Wiki](https://dev.azure.com/msazure/AzureWiki/_wiki/wikis/AzureWiki.wiki/37684/Breaking-Changes).

Please follow the link to find more details on [PR review process](https://aka.ms/SwaggerPRReview).

### CODE REVIEW

**Comment by raych1** [comment]
@sushil490023 , can you resolve `avocado` check errors?

File: specification/automation/resource-manager/Microsoft.Automation/stable/2022-07-15/python3package.json
Comment (rkmanda) [inline specification/automation/resource-manager/Microsoft.Automation/stable/2022-07-15/python3package.json:6]: The first commit needs to be the previous api version and the new changes should only be added in the subsequent commits. This is explained in detail here (If using Open API which is recommended)  [https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/208/OpenAPI-Hub-Adding-new-API-version](https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/208/OpenAPI-Hub-Adding-new-API-version) or here(For manual process) :  [https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/83/Manual-Process-Adding-new-API-version](https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/83/Manual-Process-Adding-new-API-version) . Please remove the "ARMChangesRequested" tag once you have fixed this issue.

**Comment by raych1** [comment]
@sushil490023 , can you fix `prettierCheck` error and get breaking change review approval?

@jianyexi , can you guide on the 'duplicated model' `lintDiff` error?

File: specification/automation/resource-manager/Microsoft.Automation/stable/2022-07-15/python3package.json
Comment (sushil490023) [reply specification/automation/resource-manager/Microsoft.Automation/stable/2022-07-15/python3package.json]: Hi rkmanda, Thanks for reviewing. 
I have added first commit with pervious api version swagger and committed my changes to next commit can you please review again.

**Comment by sushil490023** [comment]
Hi @raych1  I think  it is false positive breaking change As
With first of changes we were trying to add new resources to existing API version which resulted breaking changes, but now **instead of adding swagger in old API version we have created A new API version and Added swagger to the new API version.**

**Comment by raych1** [comment]
> Hi @raych1 I think it is false positive breaking change As With first of changes we were trying to add new resources to existing API version which resulted breaking changes, but now **instead of adding swagger in old API version we have created A new API version and Added swagger to the new API version.**

You're right, cross version breaking change task passed too.

**Comment by zizw123** [comment]
>             "$ref": "../../common/v1/definitions.json#/parameters/ResourceGroupNameParameter"

Try use v3 when possible

---
Refers to: specification/automation/resource-manager/Microsoft.Automation/stable/2022-08-08/python3package.json:56 in b311391. [](commit_id = b311391ed2b0ea90457e1593e4ff80b9516816a9, deletion_comment = False)

**Comment by zizw123** [comment]
>             "modelAsString": false

Any particular reason why this is false, it will be much easier to add if this is true [https://armwiki.azurewebsites.net/api_contracts/guidelines/openapi.html?q=modelAsString#oapi015-avoid-modelling-a-property-as-string-type-when-it-should-be-an-enums](https://armwiki.azurewebsites.net/api_contracts/guidelines/openapi.html?q=modelAsString#oapi015-avoid-modelling-a-property-as-string-type-when-it-should-be-an-enums)

---
Refers to: specification/automation/resource-manager/Microsoft.Automation/stable/2022-08-08/python3package.json:636 in b311391. [](commit_id = b311391ed2b0ea90457e1593e4ff80b9516816a9, deletion_comment = False)

**Comment by sushil490023** [comment]
> > ```
> >         "$ref": "../../common/v1/definitions.json#/parameters/ResourceGroupNameParameter"
> > ```
> 
> Try use v3 when possible
> 
> Refers to: specification/automation/resource-manager/Microsoft.Automation/stable/2022-08-08/python3package.json:56 in [b311391](https://github.com/Azure/azure-rest-api-specs/commit/b311391ed2b0ea90457e1593e4ff80b9516816a9). [](commit_id = [b311391](https://github.com/Azure/azure-rest-api-specs/commit/b311391ed2b0ea90457e1593e4ff80b9516816a9), deletion_comment = False)

Sure Updated Thanks :)

**Comment by sushil490023** [comment]
> > ```
> >         "modelAsString": false
> > ```
> 
> Any particular reason why this is false, it will be much easier to add if this is true https://armwiki.azurewebsites.net/api_contracts/guidelines/openapi.html?q=modelAsString#oapi015-avoid-modelling-a-property-as-string-type-when-it-should-be-an-enums
> 
> Refers to: specification/automation/resource-manager/Microsoft.Automation/stable/2022-08-08/python3package.json:636 in [b311391](https://github.com/Azure/azure-rest-api-specs/commit/b311391ed2b0ea90457e1593e4ff80b9516816a9). [](commit_id = [b311391](https://github.com/Azure/azure-rest-api-specs/commit/b311391ed2b0ea90457e1593e4ff80b9516816a9), deletion_comment = False)

Updated it to enum Thanks

**Comment by sushil490023** [comment]
@zizw123 Thanks for reviewing.

With latest commit we have also added Response for 201 Status code in API Version for swagger hybridRunbookWorkerGroup and hybridRunbookWorker

**Comment by sushil490023** [comment]
> > ```
> >         "$ref": "../../common/v1/definitions.json#/parameters/ResourceGroupNameParameter"
> > ```
> 
> Try use v3 when possible
> 
> Refers to: specification/automation/resource-manager/Microsoft.Automation/stable/2022-08-08/python3package.json:56 in [b311391](https://github.com/Azure/azure-rest-api-specs/commit/b311391ed2b0ea90457e1593e4ff80b9516816a9). [](commit_id = [b311391](https://github.com/Azure/azure-rest-api-specs/commit/b311391ed2b0ea90457e1593e4ff80b9516816a9), deletion_comment = False)

these definitions is only present in v1, Thanks

**Comment by zizw123** [comment]
> 

See https://github.com/Azure/azure-rest-api-specs-pr/blob/main/specification/common-types/resource-management/v3/types.json#L638

Will not block this PR on this, signing off, and you can take a look at these v3

**Comment by raych1** [comment]
@ArcturusZhang @msyyc , can you have a look at the SDK automation errors?

**Comment by msyyc** [comment]
Hi @sushil490023 Please check the duplicated definition: 
image

File: specification/automation/resource-manager/Microsoft.Automation/stable/2022-08-08/runbook.json
Comment (Jassingh819) [inline specification/automation/resource-manager/Microsoft.Automation/stable/2022-08-08/runbook.json:82]: Need to change this to file type.

File: specification/automation/resource-manager/Microsoft.Automation/stable/2022-08-08/runbook.json
Comment (Jassingh819) [inline specification/automation/resource-manager/Microsoft.Automation/stable/2022-08-08/runbook.json:367]: Need to change this to file type.

File: specification/automation/resource-manager/Microsoft.Automation/stable/2022-08-08/dscConfiguration.json
Comment (Jassingh819) [inline specification/automation/resource-manager/Microsoft.Automation/stable/2022-08-08/dscConfiguration.json:309]: Need to change this to file type. - https://github.com/Azure/azure-rest-api-specs/pull/3749/files

**Review by jianyexi — APPROVED** [review]

**Comment by sushil490023** [comment]
@raych1 Now ARM Manifest changes has been deployed in all the public region and we can merge this PR.

**Pull request merged**
